defender/downloaded/Contract_cToken.js

```diff
@@ -3,9 +3,24 @@ const axios = require('axios');
 const axiosRetry = require('axios-retry');
 const ethers = require('ethers');
 
+// function to calculate the delay until the next request attempt
+// returns a value specified in milliseconds
+function retryDelayFunc(retryCount) {
+  // 300 seconds total in Autotask execution to perform retries
+  // #   time   attempt
+  // 0 -   0s - initial request
+  // 1 -  40s - first retry (40s delay from initial request)
+  // 2 - 120s - second retry (80s delay from first retry)
+  // 3 - 280s - third retry (160s delay from second retry)
+  // this leaves 20s for the rest of the Autotask to execute, plus
+  // whatever time each request takes
+  const delay = (retryCount**2)*40*1000;
```

Comment:
```suggestion
  const delay = (2**retryCount)*20*1000;
```
@darian-ae I think this is what was intended. It matches the comments and allows all retries to finish before the 300.10 second autotask timeout.

Reply:
Yes, correct! Fixed!